docs/backup_workstations.rst

```diff
@@ -118,16 +115,83 @@ Then, copy the contents of the device's persistent volume to the directory using
 
 .. code:: sh
 
-  rsync -a --info=progress2 --no-specials --no-devices \
-      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup
+  sudo bash -c "rsync -a --info=progress2 --no-specials --no-devices \
+      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup && sync"
+
+.. note:: Note that the first path argument ends with a ``/``, but the second 
+          path argument should not!
 
-Once complete, unmount the TailsData partition.
+
+Once complete, unmount the TailsData partition by clicking the Eject button 
+beside its entry in the lefthand column of the file manager. When its entry is 
+no longer shown in the lefthand column, it is save to remove the 
+*Admin Workstation* USB.
 
 Repeat these steps for every device, making a new folder on the backup device
 for each device you back up.
 
 Finally, once you have completed the steps described in this section for each
-USB drive, unmount the Backup partition and store the drive somewhere safely.
+USB drive, unmount the Backup partition by clicking its Eject button. Wait until
+the Backup USB can be safely removed, and store it somewhere safely.
+
+.. note:: After the Eject button is clicked, it may be take some time before 
+          the drive can be safely removed. Wait until its entry  is removed from
+          the lefthand column of the file manager.
+
+Restoring a Workstation from a Backup
+=====================================
+
+To recreate a backed-up *Admin Workstation*, *Journalist Workstation*, or 
+*Secure Viewing Station* Tails USB,  you will need your Backup USB, a master
+Tails USB, and a fresh USB stick to be set up as the new device. You will also 
+need an airgapped machine, as the process is similar to that originally used to 
```

Comment:
Overall these docs are super-clear, thank you!

I'm a bit confused by this part; you are referring the need to have a master _and_ the new Tails device, each with a persistent volume. What's unclear to me from reading the instructions from here on is which persistent volume we're restoring to, and whether a master device is always needed.

In short, I think that part may still require some untangling and explanation upfront, to help the user understand at this point in the process what the role of each Tails drive is. The use of a term like "master" may be unfamiliar, as well.

Reply:
That's a fair point, it could do with some upfront definitions so I'm not just winging it in how I refer to each device. Will update.

Reply:
have now done so!